parallel-install/pkg/config/config_test.go

```diff
@@ -30,24 +30,16 @@ func Test_ValidateDeletion(t *testing.T) {
 	t.Run("Happy path", func(t *testing.T) {
 		fpath := filePath(t)
 		config = Config{
-			WorkersCount:   1,
-			ComponentList:  newComponentList(t),
-			KubeconfigPath: filepath.Dir(fpath),
+			WorkersCount:  1,
+			ComponentList: newComponentList(t),
+			KubeconfigSource: KubeconfigSource{
+				Path:    filepath.Dir(fpath),
```

Comment:
Is there a reason to strip the filename from the path here?

Reply:
It used to do so before the refactor when only path was provided, so I left it the same but now using `KubeconfigSource`.

Reply:
What happens if for example the user provides a path like this?
`/some/path/totally-random-filename.yaml`

Will the implementation of `KubeconfigSource` still work?